misc/multistream-select/src/protocol/mod.rs

```diff
@@ -20,47 +20,49 @@
 
 //! Contains lower-level structs to handle the multistream protocol.
 
+const MSG_MULTISTREAM_1_0: &[u8] = b"/multistream/1.0.0\n";
+const MSG_PROTOCOL_NA: &[u8] = b"na\n";
+const MSG_LS: &[u8] = b"ls\n";
+
 mod dialer;
 mod error;
 mod listener;
 
-const MULTISTREAM_PROTOCOL_WITH_LF: &[u8] = b"/multistream/1.0.0\n";
-
 pub use self::dialer::{Dialer, DialerFuture};
 pub use self::error::MultistreamSelectError;
 pub use self::listener::{Listener, ListenerFuture};
 
 /// Message sent from the dialer to the listener.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum DialerToListenerMessage<N> {
+pub enum Request<N> {
```

Comment:
We're kind of trying to maintain some backwards compatibility, so I'm not such a fan of renaming things just for renaming.

Reply:
This is internal API though, as far as I'm aware. Am I just not seeing where these types are being exported from the crate?

Reply:
Oh ok, I misremembered. Feel free to discard my comment them.